sydent/replication/peer.py

```diff
@@ -83,8 +84,22 @@ def __init__(self, sydent, server_name, pubkeys):
         self.port = 1001
 
         # Get verify key for this peer
-        key_bytes = decode_base64(self.pubkeys[SIGNING_KEY_ALGORITHM])
-        self.verify_key = signedjson.key.decode_verify_key_bytes(SIGNING_KEY_ALGORITHM + ":", key_bytes)
+
+        # Check if their key is base64 or hex encoded
+        pubkey = self.pubkeys[SIGNING_KEY_ALGORITHM]
+        try:
+            # Check for hex encoding
+            pubkey_decoded = int(pubkey, 16)
+            logger.warn("Peer %s public key is hex encoded. Please update to base64 encoding", server_name)
+        except:
```

Comment:
can you limit this to `ValueError` rather than catching everything up to and including out-of-memory errors ?